lib/webhook-proxy.js

```diff
@@ -1,14 +1,31 @@
 module.exports = ({url, port, path, logger}) => {
   try {
+    var goodUrl = true // url is considered to be true
```

Comment:
This should be `let goodUrl`

Reply:
I think this is good as **var goodUrl**  because then i'm unable to use it in next block. 😅